addNew.js

```diff
@@ -11,9 +11,13 @@ const enter = document.querySelector('.fa-level-down-alt');
 let toDoDivsAll = document.querySelectorAll('.todo-item');
 const fullList = document.querySelector('.full-list');
 const newItem = document.getElementById('new-item');
-let toDoText = document.querySelectorAll('.todo');
 let numberIndex = 0;
 let itemText = '';
+let toDoText = '';
+
+function initializeToDoText() {
```

Comment:
congratulations on implementing this function that made your codes DRIER than it was .🎉